app/CMakeLists.txt

```diff
@@ -1,8 +1,10 @@
 add_executable(bergamot-translator-app main.cpp)
 target_link_libraries(bergamot-translator-app PRIVATE bergamot-translator)
 
-add_executable(service-cli main-mts.cpp)
-target_link_libraries(service-cli PRIVATE bergamot-translator)
+if (NOT COMPILE_WASM)
```

Comment:
This change is not required because app folder is not added for wasm builds at all. [See here](https://github.com/browsermt/bergamot-translator/blob/jp/find-segfault/CMakeLists.txt#L72-L74).

Please revert changes in this file 👍

Reply:
Covered in https://github.com/browsermt/bergamot-translator/pull/44/commits/e1b74bccab2ea32c1c02f92f22c38f170ec55ae6.